addons/website/static/src/services/website_service.js

```diff
@@ -110,7 +110,7 @@ export const websiteService = {
                     return;
                 }
                 // Not all files have a dataset. (e.g. XML)
-                if (!document.documentElement.dataset) {
+                if (!Object.keys(document.documentElement.dataset).length) {
```

Comment:
I tested on chrome only, a bit quickly... It looks like opening non html files is not handle the same way across browsers:
- on chrome, the document will have an empty dataset,
- on firefox, it will not have a dataset.

As we only want to show the edition options for html documents, maybe we could test on the doctype: `document.doctype && document.doctype.name === 'html'`?

Reply:
That must be because Firefox tries to follow HTML specs and Chrome simply don't care :smile: I guess checking the doctype is a possibility but can't you use instanceof like I said to do the same? `if (document instanceof HTMLDocument) {` ? But doctype is fine too

Reply:
I would have thought it gives the same result but for some reason, it does not. I have typescript in my global node modules telling me that HTMLDocument is deprecated but I have found no concrete mention about that online
![image](https://user-images.githubusercontent.com/58561718/198073345-82bade48-b9dc-4a19-bc26-cfe49fcf5e57.png)
![image](https://user-images.githubusercontent.com/58561718/198073858-32173403-1cdf-43e3-b63a-0c2c73ca509f.png)

Reply:
Strange, ok for the doctype then of course :+1: